logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/call/usecase/GetIncomingCallsUseCase.kt

```diff
@@ -1,22 +1,52 @@
 package com.wire.kalium.logic.feature.call.usecase
 
-import com.wire.kalium.logic.feature.call.Call
+import com.wire.kalium.logic.data.conversation.ConversationDetails
+import com.wire.kalium.logic.data.conversation.ConversationRepository
+import com.wire.kalium.logic.data.notification.LocalNotificationCall
 import com.wire.kalium.logic.feature.call.CallManager
 import com.wire.kalium.logic.feature.call.incomingCalls
+import com.wire.kalium.logic.functional.flatMapFromIterable
 import com.wire.kalium.logic.sync.SyncManager
 import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.distinctUntilChanged
+import kotlinx.coroutines.flow.flatMapLatest
+import kotlinx.coroutines.flow.map
 
 interface GetIncomingCallsUseCase {
-    suspend operator fun invoke(): Flow<List<Call>>
+    suspend operator fun invoke(): Flow<List<LocalNotificationCall>>
 }
 
 internal class GetIncomingCallsUseCaseImpl(
     private val callManager: CallManager,
-    private val syncManager: SyncManager
+    private val syncManager: SyncManager,
+    private val conversationRepository: ConversationRepository
 ) : GetIncomingCallsUseCase {
 
-    override suspend operator fun invoke(): Flow<List<Call>> {
+    //TODO update UnitTests after fixing all the other TODOs
+    override suspend operator fun invoke(): Flow<List<LocalNotificationCall>> {
```

Comment:
Just a reminder that this change will require Reloaded to be updated so it doesn't break.

Reply:
exactly, this change will break the feature of incoming call

Reply:
actually no, this won't break anything in Reloaded. Cause Kotlin will replace object type from `Call` to the new one by itself and there are the same filed names (`conversationId` and `status`).